config/settings.yml

```diff
@@ -63,6 +63,7 @@ max_friends_per_hour: 60
 min_changeset_comments_per_hour: 1
 initial_changeset_comments_per_hour: 6
 max_changeset_comments_per_hour: 60
+min_changeset_comments_for_max_rate_limit: 200
```

Comment:
That's a very long name and also inaccurate because it's the maximum number of changesets we consider, not the minimum.

I'm not sure what the best name is though - maybe something like `changeset_comments_for_rate_limit` would do? It should also be moved above the other settings I think rather then intermingling it with the `per_hour` settings.

Reply:
Named it  min_changeset_comments_for_max_rate_limit because it's the minimum number of changeset comments a user should have to get the max rate limit of 60 per hour. But i agree it's long.

Reply:
@GobinathAL : can you update the pull request as suggested, please?

Reply:
@mmd-osm I've updated it.

Reply:
You don't need another commit to rename a thing you added in a previous commit. One commit is enough here.